core/src/test/resources/org/apache/calcite/test/SqlToRelConverterTest.xml

```diff
@@ -7074,17 +7114,17 @@ from dept]]>
     </Resource>
     <Resource name="plan">
```

Comment:
If we don't have a similar query in `sub-query.iq` please include this as well so that test that it actually runs.

Reply:
fixed